app/routes/__frontend/resources/developer-resources/developer-health/burnout.mdx

```diff
@@ -0,0 +1,101 @@
+---
+meta:
+  title: Burnout
+  description: A collection of guides to help with developer burnout.
+hero:
+  Hero: UndrawMeditating
+tags:
+  - memberresources
+  - memberresourcesIndex
+order: 1
+---
+
+import TextContainer from '~/components/content/TextContainer';
+import LeadText from '~/components/content/LeadText';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+Developer burnout is an experience that’s too prevalent in the tech industry, often resulting from workplace stress, a lack of psychological safety in the workplace, negative experiences or feedback, and results in exhaustion, an inability to complete work tasks effectively, a lack of desire to work or ambivalence towards work or teammates.
+
+Removing yourself from the situation might be the best way to recover from burnout. However, that’s not always an option. You can find some tips to navigate burnout in this article.
+
+</LeadText>
+
+</TextContainer>
+
+## Table of Contents
+
+<TextContainer>
+
+---
+
+## Removing Negative Influences
+
+We often find that shame influences our interactions and what we agree to. Identifying where you feel shame and how it impacts your burnout is a good first step. Let go of the guilt of needing a break, saying no, passing a project to another colleague, etc.
+
+Sometimes you must step away from the things you can no longer sustain. You may have taken on too much, or you changed your priorities. That’s ok. Learning to pause and step away from things is important. Trying to do too much will accelerate burnout.
+
+</TextContainer>
+
+<TextContainer>
+
+## Increasing Positive Influences
+
+Counteracting the negative influences in your life by investing in the people and things that bring you energy or happiness can help to navigate burnout. Find the activities that bring you joy and comfort, and incorporate those into your week. Put them on your schedule as you would work meetings or interviews to signify their importance and necessity.
+
+Sometimes, physically staying in the same place increases feelings of burnout. Try moving your workspace. As part of this, incorporating regular body movement can decrease feelings of being trapped. It might help to set some goals to walk X every week, start a 100-day push-up challenge, or set up a physical goal that holds you accountable but also accomplished.
+
+Create adequate time to wind down every day. This will also contribute to much better sleep, which is essential for burnout recovery.
+
+Consider making space in the week to focus on activities that bring you joy. This might be playing games, watching movies, hanging out with friends and family, heading to the gym, or doing any activity that gives you energy. Along with that, it’s ok and good to rest. Finding space not to do things is equally essential in replenishing your energy.
```

Comment:
Move "This might be playing games, watching movies, hanging out with friends and family, heading to the gym, or doing any activity that gives you energy. Along with that, it’s ok and good to rest. Finding space not to do things is equally essential in replenishing your energy." to line 46. Remove the first sentence as it's already on line 46.